tests/integration/test_format_txts_e2e.py

```
@@ -0,0 +1,266 @@
+    monkeypatch.setattr("format_txts.CONTACT_STORE_FILE", env["contacts_file"])
+    monkeypatch.setattr("format_txts.OUTPUT_DIR", env["output_dir"])
+    monkeypatch.setattr("pathlib.Path.home", lambda: env["export_dir"].parent)
+    
+    # Mock the run_imessage_exporter function to return our test export directory
+    async def mock_run_imessage_exporter(*args, **kwargs):
+        return env["export_dir"]
+    
+    # Patch the necessary functions
+    with patch("format_txts.run_imessage_exporter", mock_run_imessage_exporter):
+        # Run the main function with our test arguments
+        with patch("sys.argv", ["format_txts.py"] + test_args):
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)